controllers/grid/pubIds/form/PKPAssignPublicIdentifiersForm.inc.php

```diff
@@ -66,6 +73,7 @@ function fetch($request) {
 		$templateMgr->assign('pubObject', $this->getPubObject());
 		$templateMgr->assign('approval', $this->getApproval());
 		$templateMgr->assign('confirmationText', $this->getConfirmationText());
+		$templateMgr->assign('formParams', $this->getFormParams());
```

Comment:
I think it would be possible to add a new form subclass in OJS that will consider this attribute only. This is relative generic parameter and eventually it could be used in OMP (but it is not at the moment). Would you recommend to have it in the OJS subclass?